src/routes/mod_creation.rs

```diff
@@ -434,6 +434,8 @@ async fn mod_create_inner(
                 user_id: current_user.id.into(),
                 name: current_user.username.clone(),
                 role: crate::models::teams::OWNER_ROLE.to_owned(),
+                permissions: 0x1 | 0x2,
```

Comment:
For this to be usable, this must either use something like [bitflags](https://docs.rs/bitflags) or have a set of constants;  This gives no indication what 0x1 and 0x2 mean.